### ISSUE TITLE
test custom fork?

### ISSUE THREAD

**Issue by djungelola** [alistair23/AutoMower-BLE#28]
If i want to test my custom fork of automower-ble in custom_components, what is the correct syntax for requirements in manifest.json?
Default:
"requirements": ["automower-ble==0.1.33"]

Custom?:
"requirements": ["git+https://github.com/alistair23/AutoMower-BLE@main#automower-ble"],

Is this correct?

/Ola

**Comment by alistair23** [comment]
Good question. I'm not sure, I think something like that should work

**Comment by erikmol** [comment]
Yes that should work, you can see the last example here: [Custom requirements during development & testing](https://developers.home-assistant.io/docs/creating_integration_manifest/#custom-requirements-during-development--testing)
